modal/queue.py

```diff
@@ -44,6 +64,21 @@ def main():
 
         assert my_queue.get() == "some value"
         assert my_queue.get() == 123
+
+        my_queue.put(0)
+        my_queue.put(1, partition_key="foo")
```

Comment:
The `.put` method has `partition` as the parameter name, but you have `partition_key` here (and below).